packages/box/src/Box.tsx

```diff
@@ -1,33 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-type BoxDivDefault = {
-  as?: never;
-  href?: never;
-} & React.ComponentPropsWithRef<'div'>;
+type Override<T, U> = Omit<T, keyof U> & U;
```

Comment:
Should we define these in lib so that they can be used elsewhere?

Reply:
We could go by the rule of three as a rule of thumb. When some code appears for a third time it’s a good sign to refactor

Reply:
👍